meveo-admin-ejbs/src/main/java/org/meveo/service/git/GitClient.java

```diff
@@ -492,6 +502,18 @@ public boolean pull(GitRepository gitRepository, String username, String passwor
         if (!gitRepository.isRemote()) {
             throw new IllegalArgumentException("Repository " + gitRepository.getCode() + " has no remote to pull from");
         }
+        
+        MeveoModule module = meveoModuleService.list().stream()
```

Comment:
Here you will get some error because the git respository is not always linked to a module

Reply:
By the way I'd prefer that you create an HQL query in the MeveoModuleService that take the git repository id as input